resources/lang/en/posts.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+return [
+    'create'        => [
+        'title' => 'New Post',
+    ],
+    'fields'        => [
+        'name'          => 'Name',
+        'parent'        => 'Parent Entity',
+        'position'      => 'Position',
```

Comment:
We already have this "position" field defined in `entities/notes.fields.position`, so remove it from this file and change the form